### PR TITLE
Covered 3 talks from EmberATX

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

**Comment by ijlee2** [comment]
@chbonser Can you take a look if you have time? Feel free to make suggestions!

File: source/2020-04-03-the-ember-times-issue-142.md [hunk 1]
```diff
@@ -72,14 +74,15 @@ The Glimmer.js betas could pave the way for what we might want to do in Ember.js
...
+Hope everyone is well and taking care of themselves! 💞 If you have ~15 minutes to spare, you can watch a recent talk from EmberATX:
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+- [ember-printable-pages](https://www.youtube.com/watch?v=2FQivvOK-04&list=PLHCmVP3S9Z1PZp5JU6VxmUyuQjO7vrLfy) by [Chris Bonser (@chbonser)](https://github.com/chbonser)
+- [Introducing Ember Modifiers](https://www.youtube.com/watch?v=KWBOYqAEh6M&list=PLHCmVP3S9Z1PZp5JU6VxmUyuQjO7vrLfy) by [Ava Wroten (@sharpshark28)](https://github.com/sharpshark28)
+- [CI with GitHub Actions for Ember Apps](https://www.youtube.com/watch?v=Np3DdTrYmV0&list=PLHCmVP3S9Z1PZp5JU6VxmUyuQjO7vrLfy) by [Isaac Lee (@ijlee2)](https://github.com/ijlee2)
+
+In particular, please check out the new addon, [ember-printable-pages](https://forge512.github.io/ember-printable-pages/)! You can **write declarative templates** and **reuse components** to create printable pages for end users. It also supports **progressive rendering** to provide a smooth user experience when the end user needs to print _hundreds_ of pages. 💯
```
Comment (amyrlam) [inline source/2020-04-03-the-ember-times-issue-142.md]:
```suggestion
In particular, please check out the new addon, [ember-printable-pages](https://forge512.github.io/ember-printable-pages/)! You can **write declarative templates** and **reuse components** to create printable pages for end users. It also supports **progressive rendering** to provide a smooth user experience when the end user needs to print <span style="font-style: italic;">hundreds</span> of pages. 💯
```

**Review by amyrlam — APPROVED** [review]
💯

**Comment by amyrlam** [comment]
@ijlee2 @chbonser i'm going to merge this in for today's issue

**Pull request merged**